nipype/interfaces/base/core.py

```diff
@@ -517,7 +517,9 @@ def run(self, cwd=None, ignore_exception=None, **inputs):
         outputs = None
 
         try:
+            runtime = self._pre_run_hook(runtime)
             runtime = self._run_interface(runtime)
+            runtime = self._post_run_hook(runtime)
```

Comment:
@satra Are you comfortable with this change to `BaseInterface`?

Reply:
for now yes - we may have to add some prov stuff when i refactor that piece.